src/server/scripts/Northrend/IcecrownCitadel/boss_blood_queen_lana_thel.cpp

```diff
@@ -330,7 +330,7 @@ class boss_blood_queen_lana_thel : public CreatureScript
                         const Map::PlayerList& pl = me->GetMap()->GetPlayers();
                         for (Map::PlayerList::const_iterator itr = pl.begin(); itr != pl.end(); ++itr)
                             if (Player* p = itr->GetSource())
-                                if (p->IsAlive() && p != me->GetVictim() && p->GetGUID() != _offtankGUID && !p->IsGameMaster() && p->GetDistance(me) < 70.0f)
```

Comment:
Why remove the GM check here?

Reply:
The question is why include GM ? We don't do that for all bosses.

Reply:
Well if a GM is present while the fight occurs it should definitely ignore him imo.

It's usually only relevant for these kind of do-for-all-players-of-map checks

Reply:
@Nyeriah what's your opinion on this?

Reply:
Should select from threat list instead of searching the entire map, so the logic is bad to begin with

Reply:
it should bite target with highest thread which has no bloodmirror, this can be a tank if he is soloing the boss as he dont become bloodmirror if solo in instance. I think Bloodmirror chooses the Maintank and the closest target , if there is no other target no bloodmirror appears and you can be bitten as tank , but bloodmirror only choose targets in 40yars range